packages/stream_core/lib/stream_core.dart

```diff
@@ -1,6 +1,10 @@
+export 'package:dio/dio.dart';
+
 export 'src/api.dart';
 export 'src/errors.dart';
-export 'src/models.dart';
+export 'src/logger.dart';
+export 'src/platform.dart';
+export 'src/query.dart';
 export 'src/user.dart';
-export 'src/utils.dart';
+export 'src/utils.dart' hide SharedEmitterImpl, StateEmitterImpl;
```

Comment:
Are you sure we don't want these 2 to be usable in our other packages?

Reply:
These are the implementation, we don't instantiate the class using these. We have a factory in the interface which internally creates these instances.

eg:
```dart
abstract interface class MutableSharedEmitter<T> extends SharedEmitter<T> {
  /// Creates a new instance of [MutableSharedEmitter].
  ///
  /// When [replay] is greater than 0, the emitter will replay the last [replay] events
  /// to new subscribers. When [sync] is `true`, events are emitted synchronously.
  factory MutableSharedEmitter({
    int replay,
    bool sync,
  }) = SharedEmitterImpl<T>;
  ```